util/cron/nightly

```diff
@@ -524,10 +524,15 @@ mysystem("cd $chplhomedir && $make -j$num_procs $make_vars_opt third-party-try-o
 
 # if we are using python2, we cannot build the test-venv and/or chpldoc
 if ($python2 == 0) {
-  # Build chpldoc. Do not fail the build if it does not succeed. Do not send
-  # mail either.
-  print "Making $make_vars_opt chpldoc\n";
-  mysystem("cd $chplhomedir && $make -j$num_procs $make_vars_opt chpldoc", "make chapel chpldoc", 0, 0);
+
+  # linux32 currently cannot build chpldoc.  TODO: remove this when the linux32
+  # box is using a later version of Python than 3.5
+  if (!($ENV{'CHPL_NIGHTLY_TEST_CONFIG_NAME'} eq "linux32")) {
+    # Build chpldoc. Do not fail the build if it does not succeed. Do not send
+    # mail either.
```

Comment:
Based on this comment this won't cause the build to fail or send a separate mail, so we might be able to get away without this change? It's not obvious to me if we'll get some sort of warning in the regression mail, but it might be worth letting it go a day to find out. Thoughts?

Reply:
The trouble is that the particular failure we get messes with our checking.  It *thinks* it built the package okay and so the packages that depend on it *think* they can work, but really, they can't and they fail when run.  So we need to explicitly do something about it

Reply:
More explicitly, it downloads MarkupSafe 0.0.0 instead of 2.0.1 and pip doesn't yell that that's not a real version.  Probably because we're using a really old version of pip there

Reply:
Gothca